deploy/docker/Dockerfile

```diff
@@ -17,6 +15,14 @@ RUN cd /opt
 RUN wget https://raw.githubusercontent.com/intracom-telecom-sdn/nstat/master/deploy/provision.sh
 RUN chmod +x provision.sh
 RUN /bin/bash -c "./provision.sh"
-RUN cd /proc/sys/kernel/pty
-RUN echo "50000" > max
-RUN /bin/bash
\ No newline at end of file
+CMD ["service", "openvswitch-switch" "start"]
```

Comment:
I think this has no impact here, since the started openvswitch service will not persist in a subsequent container launch. In other words, the Dockerfile should include only actions that change the filesystem (e.g. software installation), and not ones that just create processes.